### PR TITLE
CDPT-2145 Update Coat of Arms

### PR DESCRIPTION
**Pull request by EarthlingDavey**

### CODE REVIEW

File: public/app/themes/justice/page.php [hunk 1]
```diff
@@ -35,7 +35,7 @@
                 </div>
 
                 <div class="print-only">
-                    <img src="<?php echo get_template_directory_uri() ?>/dist/img/logo-inv.png" alt="" title="">
+                    <img src="<?php echo get_template_directory_uri() ?>/dist/img/logo-inv-2.png" alt="" title="">
```
Comment (wilson1000) [inline public/app/themes/justice/page.php:38]: Buddy, can you change the name to `logo-tudor-crest-inverted.png`, please?

File: public/app/themes/justice/src/css/global.css [hunk 2]
```diff
@@ -1776,7 +1776,7 @@ span.view-all {
     width: 242px;
     height: 62px;
     text-indent: -9999px;
-    background: url(../img/logo.png) no-repeat;
+    background: url(../img/logo-2.png) no-repeat;
 }
```
Comment (wilson1000) [inline public/app/themes/justice/src/css/global.css:1780]: And `logo-tudor-crest.png`?

**Review by wilson1000 — COMMENTED** [review]
Great work @EarthlingDavey - just a name change please 👍

File: public/app/themes/justice/src/css/global.css (hunk 2 above)
Comment (EarthlingDavey) [reply public/app/themes/justice/src/css/global.css:1780]: Absolutely :) Done!

**Comment by EarthlingDavey** [comment]
Hey @wilson1000 👋 Thanks for your suggestion! Done, I think it needs a re-review.

**Review by wilson1000 — APPROVED** [review]
Nice ! 😄

**Pull request merged**